prow/started_and_finished.md

```diff
@@ -0,0 +1,123 @@
+# Understanding Started.json and Finished.json
```

Comment:
nit: something like metadata_artifacts.md would probably be more apt than just started and finished.
(eventually) we would also want other files documented here as well.

Reply:
Done